pkgs/development/python-modules/colcon-python-setup-py/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  colcon,
+  pytest-cov-stub,
+  pytestCheckHook,
+  setuptools,
+  scspell,
+  writableTmpDirAsHomeHook,
+}:
+
+buildPythonPackage rec {
+  pname = "colcon-python-setup-py";
+  version = "0.2.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "colcon";
+    repo = "colcon-python-setup-py";
+    tag = version;
+    hash = "sha256-N+OL0rSoWwZZioMV9JRvrQHdahE3fY7kKjfflUiRVL8=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    colcon
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov-stub
+    pytestCheckHook
+    scspell
+    writableTmpDirAsHomeHook
+  ];
+
+  pythonImportsCheck = [
+    "colcon_python_setup_py"
+  ];
+
+  disabledTestPaths = [
+    "test/test_flake8.py"
+  ];
+
+  meta = {
+    description = "Extension for colcon-core to support Python packages with the metadata in the setup.py file";
+    homepage = "http://colcon.readthedocs.io/";
```

Must have missed this first time, but this can be https

```suggestion
    homepage = "https://colcon.readthedocs.io/";
```

---

I've forgotten it. Done!